### ISSUE TITLE
about implementing local and precise datas

### ISSUE THREAD

**Issue by timqin123** [198808xc/Pangu-Weather#13]
if given more precise local collected Datas, would it be possible to reweight the parameters and thereby increases the accuracy of a specific location?   I was thinking about Fine-tuning by LoRA, but found it nearly impractical since the parameters are somewhat indeterminate and must be trained by all latitude and longitude possible.

**Comment by bkf-huawei** [comment]
I guess may you could add some new architecture onto the neural network, and fine-tune the corresponding parameters?
